### DISCUSSION TITLE
How to apply LogViewerServiceProvider depends on environments?

### DISCUSSION THREAD

**Discussion by eyedroot** [opcodesio/log-viewer#72] [Q&A] [answered]
I want to apply `LogViewerServiceProvider` depends on **environments** for example `dev`.

is it any idea!?

**Comment by arukompas** [comment, marked as answer]
Hey @binbyz 

Log Viewer is auto-discovered like many other Laravel packages. 

You can opt out of auto-discovery by following the guide here:

https://laravel.com/docs/9.x/packages#opting-out-of-package-discovery

Once you've done this, you'll need to manually register the `LogViewerServiceProvider`
